docs/api-references/user.md

```diff
@@ -0,0 +1,123 @@
+## User API
+
+### `GET` User
+```
+/api/users/
+```
+Get information about a current user login.
+
+#### Request header
+```
+  "Accept": "application/json",
+  "Authorization": "{token_type} {access_token}"
+```
+#### Sample Response
+```json
+{
+  "data": {
+    "id": 74,
+    "full_name": "dungvan2512",
+    "email": "dungvan@test.email.com",
+    "birthday": null,
+    "gender": 0,
+    "address": null,
+    "phone_number": null,
+    "image": "http://foodmarket.com/images/users/default.jpg",
+    "is_admin": 1,
+    "is_active": 0
+  },
+  "success": true
+}
+```
+
+### `Post` user
+```
+/api/users/create
```

Comment:
To create a user. user endpoint `/api/users/` with method `POST`.
You can check in this document: https://laravel.com/docs/5.4/controllers#resource-controllers